sampleapi/api.go

```diff
@@ -0,0 +1,10 @@
+package sampleapi
+
```

Comment:
// Review:
// This code looks good. It imports the fmt package, which is necessary for the Sprintf function, and the function itself is concise and easy to understand. No changes necessary.